api/DHSC.FingertipsNext.Api.IntegrationTests/DataManagement/DataManagementIntegrationTests.cs

```diff
@@ -19,6 +27,13 @@ public DataManagementIntegrationTests(CustomWebApplicationFactory<Program> facto
     {
         _factory = factory;
 
+        using var scope = _factory.Services.CreateScope();
+        var healthDbContext = scope.ServiceProvider.GetRequiredService<HealthDataDbContext>();
+        var connectionString = healthDbContext.Database.GetDbConnection().ConnectionString;
```

Comment:
I don't think it matters, because you're just using it to get a DB connection string, but this should presumably reference `DataManagementDbContext`:
```suggestion
        var dbContext = scope.ServiceProvider.GetRequiredService<DataManagementDbContext>();
        var connectionString = dbContext.Database.GetDbConnection().ConnectionString;
```